WalletWasabi.Fluent/Views/Settings/GeneralSettingsTabView.axaml

```diff
@@ -11,37 +11,37 @@
   <StackPanel Classes="settingsLayout">
 
     <DockPanel>
-      <TextBlock Text="Dark Mode" />
+      <TextBlock Text="Dark mode" />
       <ToggleSwitch IsChecked="{Binding DarkModeEnabled}" />
     </DockPanel>
 
     <DockPanel>
-      <TextBlock Text="Start Wasabi with the system" />
+      <TextBlock Text="Run Wasabi when computer starts" />
       <ToggleSwitch IsChecked="{Binding RunOnSystemStartup}" Command="{Binding StartupCommand}" />
     </DockPanel>
 
     <DockPanel>
-      <TextBlock Text="Hide Window on close" />
+      <TextBlock Text="Close Wasabi to system tray" />
```

Comment:
```suggestion
      <TextBlock Text="Minimize Wasabi to system tray" />
```

Reply:
Minimize is another thing. That means if you put Wasabi to minimize it won't be visible on the taskbar, but only in the system tray.